### PR TITLE
Updated UI

### PR DESCRIPTION
**Pull request by j3sx**

### CODE REVIEW

File: Frontend/confirm.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <!-- Favicon -->
+    <link rel="icon" href="img/urx.png" type="image/png">
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
+  <!-- Bootstrap core CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```
Comment (mikeazz) [inline Frontend/confirm.html:24]: How was this generated?

File: package.json [hunk 2]
```diff
@@ -1,21 +1,49 @@
 {
-    "name": "ronadatabase",
-    "version": "1.0.0",
-    "description": "",
+    "title": "Landing Page",
+    "name": "startbootstrap-landing-page",
```
Comment (mikeazz) [inline package.json:3]: needs update

File: package.json [hunk 3]
```diff
@@ -1,21 +1,49 @@
...
+        "email": "feedback@startbootstrap.com"
+    },
+    "license": "MIT",
+    "author": "Start Bootstrap",
+    "contributors": [
+        "David Miller (http://davidmiller.io/)"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/StartBootstrap/startbootstrap-landing-page.git"
```
Comment (mikeazz) [inline package.json:29]: needs update

**Review by mikeazz — COMMENTED** [review]
Looks good, but there seems to be a lot of minified code. Ideally it should not be pushed to the repository, and generated at build time. Let's try to make that work, so that the build runs as part of deployment to the cloud and generates those bootstrap files.

File: Frontend/confirm.html (hunk 1 above)
Comment (j3sx) [reply Frontend/confirm.html:24]: That was from the Bootstrap template that I used for the landing page. I just copied and pasted everything in the head over to that HTML file.

That can be removed.

File: package.json (hunk 2 above)
Comment (j3sx) [reply package.json:3]: This is from the Bootstrap template that I used for the landing page.

File: package.json (hunk 3 above)
Comment (j3sx) [reply package.json:29]: This is from the Bootstrap template that I used for the landing page.

File: package.json (hunk 2 above)
Comment (mikeazz) [reply package.json:3]: Right, they're dummy values. Ideally, we would fill them in with actual values for our project. Not a blocker though

**Pull request merged**